core/src/test/java/com/graphhopper/storage/AbstractGraphStorageTester.java

```diff
@@ -122,12 +122,9 @@ public void testSetTooBigDistance_435() {
         EdgeIteratorState edge1 = graph.edge(0, 1, maxDist, true);
         assertEquals(maxDist, edge1.getDistance(), 1);
 
-        // max out should NOT lead to infinity as this leads fast to NaN!
```

Comment:
This test only passed previously because there was no check that an exception is thrown at all!

Reply:
ok